internal/datacoord/compaction_trigger.go

```diff
@@ -113,6 +114,12 @@ func (cs *compactionSignal) Notify(result error) {
 	}
 }
 
+type category struct {
+	collectionID int64
+	partitionID  int64
+	channelName  string
+}
+
```

Comment:
We already have a same struct named CompactionGroupLabel, please use that instead